rest/config.go

```diff
@@ -719,8 +722,19 @@ func (config *ServerConfig) deprecatedConfigLoggingFallback() (warnings []base.D
 			warnings = append(warnings, func() {
 				base.Warnf(warningMsgFmt, `logging.["default"].LogFilePath`, "logging.log_file_path")
 			})
+
+			// Ensures existence of the default logging.["default"].LogFilePath if specified.
+			// SGCollect relies on this path to pick up the standard and rotated log files.
+			info, err := os.Stat(*config.Logging.DeprecatedDefaultLog.LogFilePath)
+			if os.IsNotExist(err) {
+				return warnings, fmt.Errorf("the specified logging.[\"default\"].LogFilePath %q does not exist",
+					*config.Logging.DeprecatedDefaultLog.LogFilePath)
+			}
 			// Set the new LogFilePath to be the directory containing the old logfile, instead of the full path.
-			config.Logging.LogFilePath = filepath.Dir(*config.Logging.DeprecatedDefaultLog.LogFilePath)
+			if err == nil && info.Mode().IsRegular() {
```

Comment:
Should we be adding to warnings[] if err is non nil (and not IsNotExist)?

Reply:
Yeah, that is good idea.